### ISSUE TITLE
owl-base has an undeclared dependency on `integers`

### ISSUE THREAD

**Issue by bcdarwin** [owlbarn/owl#463]
Indeed:
```
File "src/base/dune", line 29, characters 21-29:
29 |  (libraries bigarray integers stdlib-shims))

Error: Library "integers" not found.
Hint: try: dune external-lib-deps --missing -p owl-base @@default
```

**Comment by mseri** [comment]
It should be removed from the dune file. I think it is a spurious dependency. I thought I had already sent a PR to fix it, which version is failing with it? Maybe I only need to update the opam metadata to add integers to some old version

**Comment by mseri** [comment]
I have checked: the opam file for 0.7.0 in opam-repository has a direct dependency on integers to avoid the issue, and I have just released 0.7.1 which fixes a segfault for large mmapped matrices and drops the dependency on integers. Once it is release on opam-repository it will be available.

**Comment by mseri** [comment]
The latest version of owl, dropping the dependency, has been released
